src/test/java/com/amazon/dtasdk/v2/serialization/messages/GetUserIdSerializableTest.java

```diff
@@ -0,0 +1,132 @@
+package com.amazon.dtasdk.v2.serialization.messages;
+
+import static org.junit.Assert.*;
+import org.junit.Test;
+
+import com.amazon.dtasdk.v2.serialization.serializer.JacksonSerializer;
+import com.amazon.dtasdk.v2.serialization.serializer.SerializationException;
+//import com.amazon.dtasdk.v2.serialization.messages.InstantAccessRequest;
+
+public class GetUserIdSerializableTest {
+	private JacksonSerializer serializer = new JacksonSerializer();
+    private GetUserIdSerializableRequest request = new GetUserIdSerializableRequest();
+    private GetUserIdSerializableResponse response = new GetUserIdSerializableResponse();
+    
+    @Test
+    public void testSerialize() throws SerializationException {
+        GetUserIdSerializableRequest request = new GetUserIdSerializableRequest();
+        request.setOperation(InstantAccessOperationValue.GETUSERID);
+        request.setInfoField1("nobody@amazon.com");
+        request.setInfoField2("AMZN");
+        request.setInfoField3("1234");

```

Comment:
you probably want to put:
```
 GetUserIdSerializableRequest request = new GetUserIdSerializableRequest();
    request.setOperation(InstantAccessOperationValue.GETUSERID);
    request.setInfoField1("nobody@amazon.com");
    request.setInfoField2("AMZN");
    request.setInfoField3("1234");
```
into a separate method in either a super class, or a helper to avoid duplicate code